pkg/services/accesscontrol/resourcepermissions/service.go

```diff
@@ -209,6 +213,7 @@ func (s *Service) SetBuiltInRolePermission(ctx context.Context, orgID int64, bui
 		Resource:          s.options.Resource,
 		ResourceID:        resourceID,
 		ResourceAttribute: s.options.ResourceAttribute,
+		OnlyActionSets:    s.options.OnlyActionSets,
```

Comment:
What's the benefit of having this an option for each resource? Instead of just reading the value from the settings?

Reply:
it made the whole thing be dependant on the settings instead. as we are not pulling the cfg into the resourcepermissions